Traceback (most recent call last):
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/site-packages/nbclient/client.py", line 1319, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/site-packages/jupyter_core/utils/__init__.py", line 165, in wrapped
    return loop.run_until_complete(inner)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/asyncio/base_events.py", line 650, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/contextlib.py", line 222, in __aexit__
    await self.gen.athrow(typ, value, traceback)
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/site-packages/nbclient/client.py", line 654, in async_setup_kernel
    yield
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/site-packages/nbclient/client.py", line 709, in async_execute
    await self.async_execute_cell(
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/site-packages/nbclient/client.py", line 1062, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/ivan/miniconda3/envs/bayes_training/lib/python3.11/site-packages/nbclient/client.py", line 918, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import os 
import torch
import numpy as np
import pandas as pd
import pyro
import pyro.distributions as dist
import matplotlib.pyplot as plt

from sklearn.model_selection import train_test_split
from pyro.nn import PyroSample
from pyro.infer.autoguide import (
    AutoDiagonalNormal,
    AutoLaplaceApproximation,
)
from pyro.infer import Predictive
import torch.nn as nn
from pyro.nn import PyroModule
from intro_bayesian_ml.reliability_diagrams import reliability_diagram
from laplace import Laplace

from sklearn.preprocessing import (
    OneHotEncoder,
    RobustScaler,
    LabelEncoder,
)
from IPython.core.display import display, HTML

import seaborn as sns
#import plotly.offline as pyo
import plotly.io as pio
import plotly.graph_objects as go


from intro_bayesian_ml.utilities import (
    train_dnn_model,
    get_train_test_data_loaders,
    classifier_report,
    pyro_training_with_guide,
    filter_probs_by_threshold,
)

from intro_bayesian_ml.config import (
    get_config,
    root_dir,
)

import warnings
from imblearn.over_sampling import SMOTE

warnings.filterwarnings("ignore")
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[4], line 18[0m
[1;32m     16[0m [38;5;28;01mimport[39;00m[38;5;250m [39m[38;5;21;01mtorch[39;00m[38;5;21;01m.[39;00m[38;5;21;01mnn[39;00m[38;5;250m [39m[38;5;28;01mas[39;00m[38;5;250m [39m[38;5;21;01mnn[39;00m
[1;32m     17[0m [38;5;28;01mfrom[39;00m[38;5;250m [39m[38;5;21;01mpyro[39;00m[38;5;21;01m.[39;00m[38;5;21;01mnn[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m PyroModule
[0;32m---> 18[0m [38;5;28;01mfrom[39;00m[38;5;250m [39m[38;5;21;01mintro_bayesian_ml[39;00m[38;5;21;01m.[39;00m[38;5;21;01mreliability_diagrams[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m reliability_diagram
[1;32m     19[0m [38;5;28;01mfrom[39;00m[38;5;250m [39m[38;5;21;01mlaplace[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m Laplace
[1;32m     21[0m [38;5;28;01mfrom[39;00m[38;5;250m [39m[38;5;21;01msklearn[39;00m[38;5;21;01m.[39;00m[38;5;21;01mpreprocessing[39;00m[38;5;250m [39m[38;5;28;01mimport[39;00m (
[1;32m     22[0m     OneHotEncoder,
[1;32m     23[0m     RobustScaler,
[1;32m     24[0m     LabelEncoder,
[1;32m     25[0m )

[0;31mModuleNotFoundError[0m: No module named 'intro_bayesian_ml'

